src/grammar/function_invocation.rs

```diff
@@ -13,7 +13,7 @@ use source_gen::T;
 
 /// Looks ahead and parses a function invocation if applicable
 pub(crate) fn opt_function_invocation(p: &mut Parser) -> bool {
-    if (p.at(T![quoted_ident]) || p.at(T![unquoted_ident]))
+    if p.current().is_ident()
```

Comment:
Good start, but we might have a function such as `my_schema.aggregate` :smile:

Reply:
The following `matches!` needs to be refactored to use `is_ident()` instead of `T![quoted_ident]` and `T![unquoted_ident]`.